entities/projectile.gd

```diff
@@ -33,10 +33,16 @@ func set_can_bounce(value: bool = false) -> void:
 		$Polygon2D.set_color("#ff8426") 
 		bounce_number = 3
 
+func emit_sparks() -> void:
+	var emitter = sparks.instance()
+	emitter.position = self.position
+	get_tree().get_root().add_child(emitter)
+
```

Comment:
This will need to go into the `projectile2d.gd` file. I will delete this file from the main branch after we merge this PR.